apps/financial-aid/api/src/app/modules/municipality/models/municipalitySettings.model.ts

```diff
@@ -0,0 +1,14 @@
+import { Field, ObjectType, ID, Float } from '@nestjs/graphql'
+
+import type { MunicipalitySettings } from '@island.is/financial-aid/shared/lib'
+
+import { MunicipalityAidModal } from './municipalityAid.model'
+
+@ObjectType()
+export class MunicipalitySettingsModal implements MunicipalitySettings {
+  @Field()
+  readonly homePage?: string
```

Comment:
Are we sure about this hierarchy? 

Now the object structure is: 
```
{
  id
  name
  settings {
     homePage
     aid {
        ....
     }
  }
}
```

Why not:
```
{
  id
  name
  homePage
  aid {
   ....
  }
}
```

Reply:
At the time this decision was made, since we were not sure how much information each municipality needed and therefore we knew that it would defiantly change, so we made settings obj where we can add and remove easily, however now I have destroy that with all my models... so I am open for "rethinking" the structure

Reply:
Do you want to decide this later and keep it like this for now or keep this structure or go for the suggested one? 😄